docs/essentials/datetime/EnumerateTimeZones.md

```diff
@@ -1 +1,25 @@
+# How to: Enumerate Time Zones Present on a Computer
+
+Successfully working with a designated time zone requires that information about that time zone be available to the system. For example, the Windows operating system stores this information in the registry. However, although the total number of time zones that exist throughout the world is large, the registry contains information about only a subset of them. In addition, the registry itself is a dynamic structure whose contents are subject to both deliberate and accidental change. As a result, an application cannot always assume that a particular time zone is defined and available on a system. The first step for many applications that use time zone information applications is to determine whether required time zones are available on the local system, or to give the user a list of time zones from which to select. This requires that an application enumerate the time zones defined on a local system. 
```

Comment:
> For example, the Windows operating system stores this information in the registry.

This discussion doesn't seem appropriate for essentials.These topics should either be OS agnostic or provide information for all relevant OSes.